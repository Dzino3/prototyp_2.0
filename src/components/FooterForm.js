import React from 'react'
import './style/footerStyle.css';

function FooterForm({t}) {
  return (
    <div className='footer'>
        <div className='footerLinks'>
        <a href='https://www.deere.de/de/index.html' target="blank">{t('footerLabel1')}</a>
        <a href='https://www.deere.com/en/our-company/contact-us/' >{t('footerLabel2')}</a>
        <a href='https://www.deere.de/de/forms/digital-news-service/' target="blank">{t('footerLabel3')}</a>
        <a href='https://www.deere.de/de/impressum/' target="blank">{t('footerLabel4')}</a>
        <a href='https://www.deere.de/de/ersatzteile-und-service/ersatzteile/suche-nach-teilen-h%C3%A4ufig-gestellte-fragen/' target="blank">{t('footerLabel5')}</a>
        <a href='https://www.deere.de/de/datenschutzerkl%C3%A4rung/' target="blank">{t('footerLabel6')}</a>
        <a href='https://www.deere.com/en/privacy-and-data/terms/index.html' target="blank">{t('footerLabel7')}</a>
        <a href='https://www.deere.de/de/ersatzteile-und-service/service/' target="blank">{t('footerLabel8')}</a>
      </div>
      <p className='copyright'>{t('footerLabel9')}</p>
    </div>
  )
}

export default FooterForm
