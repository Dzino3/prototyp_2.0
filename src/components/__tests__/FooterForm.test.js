import FooterForm from '../FooterForm';
import React from "react";
import renderer from 'react-test-renderer';


describe(`Rendering FooterForm`, () => {

    test('should render FooterForm', function () {
        let tree = renderer.create(<FooterForm/>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})