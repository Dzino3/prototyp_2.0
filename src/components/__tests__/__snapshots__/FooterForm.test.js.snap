// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering FooterForm should render FooterForm 1`] = `
<div
  className="footer"
>
  <div
    className="footerLinks"
  >
    <a
      href="https://www.deere.de/de/index.html"
      target="blank"
    >
      Home
    </a>
    <a
      href="https://www.deere.com/en/our-company/contact-us/"
    >
      Contact us
    </a>
    <a
      href="https://www.deere.de/de/forms/digital-news-service/"
      target="blank"
    >
      Newsletter
    </a>
    <a
      href="https://www.deere.de/de/impressum/"
      target="blank"
    >
      Impressum
    </a>
    <a
      href="https://www.deere.de/de/ersatzteile-und-service/ersatzteile/suche-nach-teilen-h%C3%A4ufig-gestellte-fragen/"
      target="blank"
    >
      FAQ
    </a>
    <a
      href="https://www.deere.de/de/datenschutzerkl%C3%A4rung/"
      target="blank"
    >
      Privacy and Data
    </a>
    <a
      href="https://www.deere.com/en/privacy-and-data/terms/index.html"
      target="blank"
    >
      Terms of Use
    </a>
    <a
      href="https://www.deere.de/de/ersatzteile-und-service/service/"
      target="blank"
    >
      Services
    </a>
  </div>
  <p
    className="copyright"
  >
    Copyright Â© 2022 Deere & Company. All Rights Reserved.
  </p>
</div>
`;
